internal/pkg/cli/dialog/create_template.go

```diff
@@ -131,6 +131,14 @@ func (d *createTmplDialog) ask() (createTemplate.Options, error) {
 		return d.out, err
 	}
 
+	// Ask for user inputs
+	objectInputs, allInputs, err := d.askTemplateInputs(opts, d.selectedBranch, d.selectedConfigs)
+	if err != nil {
+		return d.out, err
+	}
+	objectInputs.setTo(d.out.Configs)
```

Comment:
Od riadku `128` -` // Ask for new ID for each config and row`, ziskame `d.Configs` -> zoznam konfiguracii, ktore maju byt sucastou sablony + ich `templateId`.

A tu sa kazdej `ConfigDef/RowDef` nastavia aj kluce, ktore sa maju nahradit za `input`.